ui/src/app/views/concept-add-modal/component.spec.tsx

```diff
@@ -0,0 +1,134 @@
+import {mount} from 'enzyme';
+import * as React from 'react';
+
+import {ConceptSetsApiStub} from 'testing/stubs/concept-sets-api-stub';
+import {DomainCountStubVariables, ConceptStubVariables} from 'testing/stubs/concepts-api-stub';
+import {registerApiClient} from 'app/services/swagger-fetch-clients';
+import {waitOneTickAndUpdate} from 'testing/react-test-helpers';
+import {ConceptAddModal} from './component';
+import {ConceptSetsApi} from 'generated/fetch/api';
+import {WorkspaceAccessLevel} from 'generated';
+import {currentWorkspaceStore} from 'app/utils/navigation';
+import {WorkspacesServiceStub} from 'testing/stubs/workspace-service-stub';
+
+
+describe('ConceptAddModal', () => {
+  let props;
+  let conceptSetsApi: ConceptSetsApiStub;
+  const stubConcepts = ConceptStubVariables.STUB_CONCEPTS;
+  const selectedDomain = DomainCountStubVariables.STUB_DOMAIN_COUNTS[0];
+
+  const component = () => {
+    return mount(<ConceptAddModal {...props}/>);
+  };
+
+  beforeEach(() => {
+    props = {
+      onSave: () => {},
+      onClose: () => {},
+      selectedConcepts: stubConcepts,
+      selectedDomain: selectedDomain
+    };
+
+    conceptSetsApi = new ConceptSetsApiStub();
+    registerApiClient(ConceptSetsApi, conceptSetsApi);
+  });
+
+  it('gets concept set list filtered by domain selected', async () => {
```

Comment:
Nit: I don't know if this test is actually talking about the concept set list it gets? To test that I assume it would need to go into Add to Existing Set and check the values there? This just looks like `it picks up and displays text based on the domain selected`?